frontend/assets/javascripts/src/utils/form/Password.js

```diff
@@ -79,9 +70,9 @@ function ($, bean, component) {
 
         require(['js!zxcvbn'], function() {
 
-            var $passwordStrengthIndicatorTemplate = self.$passwordStrengthIndicatorTemplate();
-
-            $passwordStrengthIndicatorTemplate.insertAfter(self.getElem('PASSWORD_STRENGTH_INPUT'));
+            utilsHelper.toArray(document.querySelectorAll('.' + self.classes.STRENGTH_INDICATOR)).map(function (elem) {
+                elem.classList.toggle('is-hidden');
+            });
```

Comment:
Probably worth the discussion. Where I'm coming from is that as we are IE9+ we already have enough stuff natively that we probably don't need bean/bonzo. I was happy with just moving towards that one component at a time (even if that means loading bonzo/bean as well for a while), but if it looks it's a small enough task to do and we're all happy with the vanilla JS approach then that's great.

That said if we think that bean/bonzo are still adding value then we should keep them. As Matt says the syntactic sugar can be nicer.